projects/plugins/jetpack/extensions/blocks/ai-assistant/ai-assistant.php

```diff
@@ -28,7 +25,7 @@ function register_block() {
 		|| ! ( new Status() )->is_offline_mode()
 	) {
 		Blocks::jetpack_register_block(
-			BLOCK_NAME,
+			__DIR__,
```

Comment:
Not the biggest concern, but I find it a bit sensitive, as it is possible that at some point we will have a block that will be renamed, or the folder and we might have a mismatch

Reply:
Having a matching folder name is already a requisite, I believe. We could make sure to document it and add a check in the build process if we feel that's a concern.